code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1891,7 +1891,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/monkeysdelight/New()
 	..()
-	reagents.add_reagent(NUTRIMENT, 10)
+	reagents.add_reagent(NUTRIMENT, 15)
```

Comment:
They're a pest to make for only 10 nutriment and +5 nutriment is nothing. that's not even an additional 4noraisins